apps/artemis-web/utils/fetch-data.ts

```diff
@@ -144,6 +145,10 @@ export const sendHijackData = async (
     body: JSON.stringify(reqData),
   });
 
+  if (res.status === 403) {
+    toast("You do not have permission for this action!");
```

Comment:
what does `toast` output exactly? is it a popup, sth on console or sth else?

Reply:
It is a notification (like alert box) on the top right of the screen :)